guides/doc-Upgrading_and_Updating/topics/cloning_satellite_server.adoc

```diff
@@ -34,8 +34,8 @@ Ensure that you understand the following terms:
 
 To clone {ProjectServer}, ensure that you have the following resources available:
 
-* A minimal install of Red{nbsp}Hat Enterprise Linux 7 server to become the target server.
-Do not install Red{nbsp}Hat Enterprise Linux 7 software groups, or third-party applications.
+* A minimal install of {RHEL} 7 server to become the target server.
```

Comment:
Use `RHELServer` attribute.

Reply:
Here I wasn't sure about RHEL 7 Server vs RHELServer 7.

@Lennonka @melcorr any thoughts on what the correct version is here?

Reply:
🤞 that it's RHEL Server 7.